controllers/funding/past-grants.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const injectHeroImage = require('../../middleware/inject-hero');
+
+/**
+ * Pre-set GrantNav Facets
+ * json_query format, used to pre-select The Big Lottery Fund
+ */
+const grantNavFacets = {
```

Comment:
Do you think it's worth having a test for this that submits the form via our site to ensure Grantnav don't change their parameter names without us knowing?

Reply:
Good shout